package.json

```diff
@@ -19,7 +19,6 @@
     "devDependencies": {
         "@types/node": "^20.6.3",
         "@types/wait-on": "^5.3.1",
-        "playwright": "^1.32.2",
```

Comment:
This still needs to be here, it's used by /scripts/test-samples.ts which runs in CI